### ISSUE TITLE
Add combined building loss for wind, wave and surge

### ISSUE THREAD

**Issue by navarroc** [IN-CORE/pyincore#276]
Add building structural and content loss that handles wind, surge-wave and flood damage.

total loss = frame loss + roof loss + content loss + flooring/foundation loss
